src/test/java/io/phdata/streamliner/schemadefiner/MySQLTestContainerTest.java

```diff
@@ -98,8 +98,8 @@ public void test1StreamlinerSchemaCommand() throws FileNotFoundException {
         assertEquals(((Map<String, Object>) ingestConfigFile.get("destination")).get("type"), ((Map<String, Object>) outputConfigFile.get("destination")).get("type"));
 
         Map<String, Object> table = ((List<Map<String, Object>>) outputConfigFile.get("tables")).get(0);
-        assertEquals(table.get("type"), "Snowflake");
-        assertEquals(table.get("sourceName"), "Persons");
+        assertEquals("Snowflake", table.get("type"));
+        assertEquals("Persons", table.get("sourceName"));
```

Comment:
first field is expected and second field is actual. just reversed the values in some of the test cases.